lighthouse-core/lib/large-javascript-libraries/library-suggestions.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * @license Copyright 2020 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+
+'use strict';
+
```

Comment:
a fileoverview would be good for this file too. Unclear where it comes from and what could be added, especially if anyone starts wanting to contribute to this audit.

(though maybe filing a follow up issue for adding a `readme` to `lib/large-javascript-libraries` would be a better idea for documenting this and running `generate-bundlephobia-database.js`?)

Reply:
sounds good, added a fileoverview

Reply:
I think @brendankenny might have been hoping for a fileoverview in *this* file to explain how these suggestions were decided, what criteria we might use to add new ones, etc.

Maybe the description could be brief and link to the doc you produced where this discussion took place?

Reply:
I think this is the last thing and then LGTM

Reply:
Sg just pushed these changes